### ISSUE TITLE
Documentation for Using Plugins in Solar2D Native needs to be updated with correct procedure

### ISSUE THREAD

**Issue by enigmathix** [coronalabs/corona-docs#116]
https://docs.coronalabs.com/native/ios/usingPlugins.html states:

To use plugins with Solar2D Native builds through Xcode, you must include them in build.settings just like you would for Simulator builds, for example:

```
settings =
{
    plugins =
    {
        ["plugin.openssl"] =
        {
            publisherId = "com.coronalabs"
        },
    }, 
}
```

That just doesn't work that way anymore. This generates an error:
ERROR: plugin could not be validated: plugin.openssl (com.coronalabs)
ERROR: Activate plugin at: https://marketplace.coronalabs.com/plugin/com.coronalabs/plugin.openssl
ERROR: exiting due to plugins above not being activated.

which points to an old coronalabs url and not a way to activate the plugin.

**Comment by enigmathix** [comment]
This is what needs to go in the documentation: https://discord.com/channels/721785436195782677/724637134115438714/805127979742003272

**Comment by sekodev** [comment]
Is this issue fixed with the latest build - [2023.3686](https://github.com/coronalabs/corona/releases/tag/3686)?
From release notes: `Native: Fixes for Downloading Solar2D Plugins (#503)`

**Comment by scottrules44** [comment]
> Is this issue fixed with the latest build - [2023.3686](https://github.com/coronalabs/corona/releases/tag/3686)?
> From release notes: `Native: Fixes for Downloading Solar2D Plugins (#503)`

Yes this is fixed with 2023.3686
